### ISSUE TITLE
[Feature Req] Permission folders (website QoL)

### ISSUE THREAD

**Issue by JpegIsMe** [LuckPerms/LuckPermsWeb#463]
### Description

An optional folder feature to help organize permissions in the case where one group is filled with many permissions.
This would be exclusively for the website (/lp editor)

### Proposed Behaviour

Adding an optional grouping/folder method for permissions would help a lot with clutter in the case where one of your groups/ranks is filled to the brim with permissions.
This would not affect LP's main behaviour whatsoever, and would just be there as an optional feature to keep users organized in perm management.
(Adding an option to the `/lp [User/Group] set` command to add that permission in a specific folder would be a plus, but not necessary.)

### Extra Details

(Concept Image)
![image](https://github.com/LuckPerms/LuckPerms/assets/38381978/1e4e4ceb-e05c-4cf3-967e-2e5ab815ad2d)
Please consider this! It would make a lot of our lives easier!

**Comment by Turbotailz** [comment]
Yeah this is something I have been planning to do with the rebuild I'm looking to do :)
